server/app.js

```diff
@@ -27,8 +35,15 @@ app.use(logger('dev'));
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
-app.use(express.static(path.join(__dirname, 'public')));
-app.use(express.static(path.join(__dirname, '..', 'frontend', 'build')));
+
+// Serving a frontend
+if (process.env.NODE_ENV === 'production') {
+  app.use(express.static(path.join(__dirname, '..', 'frontend', 'build')));
+} else if (doesBuildDirExist) { // only serve build if exists. Explicit for code readability.
+  app.use(express.static(path.join(__dirname, '..', 'frontend', 'build')));
```

Comment:
aren't these the exact same lines..?

Reply:
How about I add
```
app.use(express.static(path.join(__dirname, 'server', 'public')));
```

Reply:
sure, idk what for though.
i still think you should use an or operator, cuz that's literally the whole point of the or operator's existence